sagemaker_edge_manager/sagemaker_edge_example/sagemaker_edge_greengrass_example.ipynb

```diff
@@ -0,0 +1,1202 @@
+{
```

Comment:
<p>Preferred terminology is "Choose" instead of "Click" since not not all users will be using a mouse.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='82'/>

Reply:
@massi-ang see above, please switch "click" to "choose".